components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/WebsocketInboundHandler.java

```diff
@@ -158,6 +160,11 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                 } else {
                     req.setUri(uri); // Setting endpoint appended uri
                 }
+
+                if (!Strings.isNullOrEmpty(token)) {
+                    ((FullHttpRequest) msg).headers().set(APIMgtGatewayConstants.WS_JWT_TOKEN_HEADER, token);
+
```

Comment:
Please remove extra line

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/5519/commits/55935bebfa203b2967f9ba165a730014a431e2d2